go-controller/pkg/ovn/master.go

```diff
@@ -127,14 +127,16 @@ func (oc *Controller) Start(kClient kubernetes.Interface, nodeName string) error
 func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 	// The gateway router need to be connected to the distributed router via a per-node join switch.
 	// We need a subnet allocator that allocates subnet for this per-node join switch. Use the 100.64.0.0/16
-	// or fd98::/64 network range with host bits set to 3. The subnetallocator will start allocating subnet that has upto 6
-	// host IPs)
-	joinSubnet := config.V4JoinSubnet
+	// or fd98::/64 network range with host bits set to 3. The subnetallocator will allocate a subnet that
+	// has up to 6 host IPs)
+	if config.IPv4Mode {
```

Comment:
ignorant question, this can be a combination that means dual-stack ? 
ipv4 = ipv6 = true

previous code seems to default to ipv4, does it means we don't default anymore?

Reply:
When we did single-stack IPv6, the way it worked was that everything assumed IPv4, unless `config.IPv6Mode` was set. When we added dual-stack, we pondered the best way to expose that, and ended up just adding `config.IPv4Mode`. So in single-stack, exactly one of those two will be set, and in dual-stack, both of them will be.